bindings/cs/MongoDB.Libmongocrypt/OperatingSystemHelper.cs

```diff
@@ -0,0 +1,57 @@
+﻿/*
+ * Copyright 2020–present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#if !NET452
+using System;
+using System.Runtime.InteropServices;
+#endif
+
+namespace MongoDB.Libmongocrypt
+{
+    internal enum OperatingSystemPlatform
+    {
+        Windows,
+        Linux,
+        MacOS
+    }
+
+    internal static class OperatingSystemHelper
+    {
+        public static OperatingSystemPlatform CurrentOperationSystem
```

Comment:
rename to `CurrentOperatingSystem`

global search for "operationsystem" and fix comments as well.

Reply:
Missed it, done